stress/shell/src/main/java/alluxio/stress/cli/fuse/FuseIOBench.java

```diff
@@ -153,30 +153,34 @@ public void prepare() throws Exception {
           "Single-node Fuse IO stress bench doesn't support RemoteRead or ClusterRead."
       ));
     }
-    // find 0-based id, and make sure directories and job workers are 1-to-1
     File[] jobWorkerDirs = localPath.listFiles();
     if (jobWorkerDirs == null) {
       throw new IOException(String.format(
           "--local-path %s is not a valid path for this bench. Make sure using the correct path",
               mParameters.mLocalPath
       ));
     }
+    if (!mBaseParameters.mDistributed) {
```

Comment:
Previously single-node mode will ask for number of live job worker. This is unnecessary and will break the test if the number of job worker is not 1.